depthai.py

```diff
@@ -429,7 +432,8 @@ def reset_process_wd():
             os._exit(10)
 
     for _, nnet_packet in enumerate(nnet_packets):
-        entries_prev = decode_nn(nnet_packet)
+        nnet_prev["nnet_source"] = nnet_packet
```

Comment:
@VanDavv  nnet_source is used here, holding the shared_ptr from cpp backend from going out of scope and deallocating the object

Reply:
Ok, got it, so we need to save it somewhere so it won't be deleted - thanks!